platform/client-admin/src/pages/ViewRecipientPage.tsx

```diff
@@ -46,7 +46,7 @@ export const ViewRecipientPage: React.FC<RouteComponentProps<IUpdateMatchParams>
                         postcode: model.postcode,
                         firstName: model.firstName,
                         lastName: model.lastName,
-                        prefferedProducts: model.prefferedProducts,
+                        // prefferedProducts: model.prefferedProducts,
```

Comment:
see https://github.com/aerogear/OpenVolunteerPlatform/pull/61/files#r424442326 

But I am thinking, maybe preferredProducts can just be labels - e.g `rice, bread` etc this way a recipient can declare them without having to think about `products` availability. Needs more thoughts, but I am commenting it for now.

Reply:
Yep.. I see this as simply comments section. Recipient can just put unstructured data what he needs that volunteer will read

Reply:
Exactly. Let me quick update the model in another commit.